comments/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import *
+
+urlpatterns = [
+    path('create/', CreateComment.as_view(), name="follow_stream"),
+    path('delete/', DeleteComment.as_view(), name="posts_by _stream"),
```

Comment:
Why are the names follow_stream and posts_by_stream?

Reply:
corrected